changelogs/fragments/8199-added-usb-support-to-proxmox-module.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - "proxmox_kvm - adds``usb`` parameter for setting usb devices on proxmox kvm vms (https://github.com/ansible-collections/community.general/pull/8199)"
```

According to https://en.wikipedia.org/wiki/Kernel-based_Virtual_Machine KVM is written upper-case. Also VM is an acronym, so it needs to be capitalized as well (https://docs.ansible.com/ansible/devel/dev_guide/style_guide/spelling_word_choice.html#acronyms). And then there's a period missing :)
```suggestion
  - "proxmox_kvm - adds``usb`` parameter for setting usb devices on proxmox KVM VMs (https://github.com/ansible-collections/community.general/pull/8199)."
```